docs/variants/clevo_nv41/openness_analysis.md

```diff
@@ -0,0 +1,70 @@
+# Intro
```

Comment:
@IgorBagnucki This document is not added to the mkdocs.yml. So it won't be available in the navigation toolbar.

Reply:
@macpijan  Should be fixed now